app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -75,7 +79,25 @@
     ImageView mDrawerImage;
     ImageView mDrawerUserPicture;
     private ActionBarDrawerToggle mDrawerToggle;
-    private MenuItem drawerItemToNavigateAfterSignIn = null;
+
+    int drawerItemIdToNavigateAfterSignIn = INVALID_ITEM_ID;
```

Comment:
Was there a specific reason that this is part of the PR? Feels like this is a separate change that blows up the diff.